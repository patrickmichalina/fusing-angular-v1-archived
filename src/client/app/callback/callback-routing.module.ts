import { CallbackComponent } from './callback.component'
import { NgModule } from '@angular/core'
import { RouterModule } from '@angular/router'

@NgModule({
  imports: [
    RouterModule.forChild([
      {
        path: '',
        component: CallbackComponent
      }
    ])
  ],
  exports: [RouterModule]
})
export class CallbackRoutingModule {}
