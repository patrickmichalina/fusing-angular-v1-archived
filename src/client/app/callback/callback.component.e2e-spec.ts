import { baseUrl, browser } from '../../../../tools/test/jest.e2e-setup'

describe('Callback Page', () => {
  it('should have title', async () => {
    const page = browser.goto(`${baseUrl}/callback`)

    const text = await page.evaluate(() => document.title).end()

    expect(text).toContain('Callback')
  })
})
