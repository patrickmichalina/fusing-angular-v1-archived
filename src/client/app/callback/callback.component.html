Logging you in...
<mat-spinner></mat-spinner>