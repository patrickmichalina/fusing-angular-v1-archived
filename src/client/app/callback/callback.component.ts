import { ChangeDetectionStrategy, Component } from '@angular/core'

@Component({
  selector: 'pm-callback',
  templateUrl: './callback.component.html',
  styleUrls: ['./callback.component.css'],
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class CallbackComponent {
  constructor() {
    // if (ps.isBrowser) {
    //   auth.login().subscribe()
    // }
  }
}
