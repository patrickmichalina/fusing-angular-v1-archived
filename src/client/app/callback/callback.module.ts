import { CallbackRoutingModule } from './callback-routing.module'
import { CallbackComponent } from './callback.component'
import { NgModule } from '@angular/core'
import { SharedModule } from '../shared/shared.module'

@NgModule({
  imports: [CallbackRoutingModule, SharedModule],
  declarations: [CallbackComponent],
  exports: [CallbackComponent]
})
export class CallbackModule {}
