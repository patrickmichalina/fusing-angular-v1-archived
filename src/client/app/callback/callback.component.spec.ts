import { CallbackComponent } from './callback.component'
import { TransferState } from '@angular/platform-browser'
import { async, ComponentFixture, TestBed } from '@angular/core/testing'
import { Component } from '@angular/core'
import { CallbackModule } from './callback.module'
import { AppTestingModule } from '../../../testing/app-testing.module'
import { ResponseService } from '../shared/services/response.service'

@Component({
  selector: 'test-component',
  template: '<pm-callback></pm-callback>'
})
class TestComponent {}

describe(CallbackComponent.name, () => {
  let fixture: ComponentFixture<TestComponent>

  beforeEach(
    async(() => {
      TestBed.configureTestingModule({
        imports: [CallbackModule, AppTestingModule.forRoot()],
        declarations: [TestComponent],
        providers: [ResponseService, TransferState]
      }).compileComponents()
    })
  )

  beforeEach(
    async(() => {
      fixture = TestBed.createComponent(TestComponent)
    })
  )

  afterEach(
    async(() => {
      TestBed.resetTestingModule()
    })
  )

  it(
    'should compile',
    async(() => {
      fixture.detectChanges()
      expect(fixture.componentInstance).toBeDefined()
      expect(fixture).toMatchSnapshot()
    })
  )
})
