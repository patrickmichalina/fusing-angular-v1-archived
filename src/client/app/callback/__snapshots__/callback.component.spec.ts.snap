// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CallbackComponent should compile 1`] = `
<test-component>
  <pm-callback>
    callback
  </pm-callback>
</test-component>
`;
